File: python/Course.py
class Course(): 
    
    def __init__(self, nama_matkul = ""):
        self.nama_matkul = nama_matkul
    
    def setNama_matkul(self, nama_matkul):
        self.nama_matkul = nama_matkul
    
    def getNama_matkul(self):
        return self.nama_matkul

File: c++/Program/Course.cpp
#include "header.hh"

Course::Course(){
    nama_matkul = "";
}

Course::Course(string nama_matkul){
    this->nama_matkul = nama_matkul;
}

void Course::setNamaMatkul(string nama_matkul){
    this->nama_matkul = nama_matkul;
}
string Course::getNamaMatkul(){
    return this->nama_matkul;
}

Course::~Course(){

}